packages/cli/src/services/frontend.service.ts

```diff
@@ -0,0 +1,65 @@
+import { Service } from 'typedi';
+import uniq from 'lodash/uniq';
+import { createWriteStream } from 'fs';
+import { mkdir } from 'fs/promises';
+import path from 'path';
+
+import { GENERATED_STATIC_DIR } from '@/constants';
+import { CredentialsOverwrites } from '@/CredentialsOverwrites';
+import { CredentialTypes } from '@/CredentialTypes';
+import { LoadNodesAndCredentials } from '@/LoadNodesAndCredentials';
+
+@Service()
+export class FrontendService {
```

Comment:
will move all fronted-settings out of `Server.ts` and into this class in a separate PR.

Reply:
I can take this over as well if you like.